exporter/rabbitmqexporter/metadata.yaml

```
@@ -0,0 +1,15 @@
+type: rabbitmq
+scope_name: otelcol/rabbitmq
+
+status:
+  class: exporter
+  stability:
+    development: [traces, metrics, logs]
+  distributions: [contrib]
```

your component won't be in contrib right away, it's defined here:
https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml
```suggestion
  distributions: []
```